# 2017-04-03: Siva Prasad Meduri <siva.meduri@nxp.com>
# Patch to add macro IP_MULTICAST_ALL
--- a/mcproxy/src/utils/mc_socket.cpp
+++ b/mcproxy/src/utils/mc_socket.cpp
@@ -38,6 +38,8 @@
 #include <unistd.h>
 
 #ifndef __GLIBC__
+#define IP_MULTICAST_ALL 49
+
 #include "sourcefilter.cpp"
 #endif /* __GLIBC__ */
 
